# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/ipc/PRDD.ipdl
# Commit: d158b20d9a65
# Full Hash: d158b20d9a65cd05dd69a586473216b800523083
# Author: Dan Glastonbury <dan.glastonbury@gmail.com>
# Date: 2020-10-21 09:51:28
# Regressor Bug: 1595994
# File Overlap Count: 1
# Description:
#   Bug 1595994 - PD: Allow WMF decoding inside RDD process. r=mattwoodrow
#   
#   This assumes that access to Windows WMF and DirectX is allowed from RDD process.
#   
#   Depends on D91699
# ==============================================================================

diff -r d0c9c64867cc -r d158b20d9a65 dom/media/ipc/PRDD.ipdl
--- a/dom/media/ipc/PRDD.ipdl	Tue Oct 20 23:26:49 2020 +0000
+++ b/dom/media/ipc/PRDD.ipdl	Tue Oct 20 23:26:52 2020 +0000
@@ -44,7 +44,8 @@
 
   async UpdateVar(GfxVarUpdate var);
 
-  async InitVideoBridge(Endpoint<PVideoBridgeChild> endpoint);
+  async InitVideoBridge(Endpoint<PVideoBridgeChild> endpoint,
+                        ContentDeviceData contentDeviceData);
 
   async GetUntrustedModulesData() returns (UntrustedModulesData? data);
 